src/minirest.erl

```diff
@@ -211,7 +211,7 @@ format_msg(Message)
   when is_atom(Message);
        is_binary(Message) -> Message;
 
-format_msg(Message) when is_tuple(Message) ->
+format_msg(Message) when is_tuple(Message); is_list(Message)  ->
```

Comment:
Great. Since you are here.
the `format_msg` function can be implemented as

```
format_msg(Message) ->
    try unicode:characters_to_binary(Msg)
    catch error : badarg -> iolist_to_binary(io_lib:format("~0p", [Message]))
    end.
```